hail/python/test/hailtop/utils/test_utils.py

```diff
@@ -126,3 +127,59 @@ def test_parse_docker_image_reference():
     assert x.digest is None
     assert x.name() == 'us-docker.pkg.dev/my-project/my-repo/test-image'
     assert str(x) == 'us-docker.pkg.dev/my-project/my-repo/test-image'
+
+
+def test_grouped_size_0_groups_9_elements():
+    try:
+        actual = list(grouped(0, [1,2,3,4,5,6,7,8,9]))
```

Comment:
This should throw an error. So there is no `actual` or `expected` values here. It should just be this:

```python3
try:
    list(grouped(0, [1,2,3,4,5,6,7,8,9]))
except ValueError:
    pass
else:
    assert False
```